<?xml version="1.0" encoding="UTF-8"?>
<?xml-stylesheet type="text/css" href="../../resources/stylesheets/wasmsgxml.css"?>
<TMSSource name="TivoliMessages" tmsVersion="1.0" xml:lang="en">
<!-- CMVC MSG File Name = SERV1/ws/code/security.sas/src/com/ibm/ws/security/auth/kerberos/krbsecurity.nlsprops -->
<!-- DO NOT EDIT THIS FILE - This file was generated by the XML/Html & Property emitter -->
<!-- BEGIN MESSAGES -->
<Message ID="SECJ9333W" severity="W" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.cannot.resetkeytab" varFormat="Java">
    security.auth.kerberos.cannot.resetkeytab=SECJ9333W: No se puede restablecer una propiedad del sistema KRB5_KTNAME (archivo keytab de Kerberos) en {0} porque ya está establecida para {1}. El tiempo de ejecución aún utiliza el archivo keytab de Kerberos {2}
  </MsgText>
  <Explanation>
    La propiedad del sistema para KRB5_KTNAME (archivo keytab de Kerberos) no permite el restablecimiento.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SECJ9329E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.init.crednotforwardable" varFormat="Java">
    security.auth.kerberos.init.crednotforwardable=SECJ9329E: La credencial {0} no se puede reenviar para el nombre de servicio GSS del destino {1} en el reino {2}
  </MsgText>
  <Explanation>
    La credencial no es reenviable.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SECJ9317E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.CredInvalid" varFormat="Java">
    security.auth.kerberos.CredInvalid=SECJ9317E: La credencial no es válida.
  </MsgText>
  <Explanation>
    Excepción inesperada. La causa no se puede determinar inmediatamente.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SECJ9204E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.CredExpired" varFormat="Java">
    security.auth.kerberos.CredExpired=SECJ9204E: El estado de la credencial no es válido.
  </MsgText>
  <Explanation>
    Se ha intentado acceder a una credencial caducada.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SECJ9316E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.GSSException" varFormat="Java">
    security.auth.kerberos.GSSException=SECJ9316E: Se ha producido una excepción GSSexception inesperada al intentar ejecutar el método {0} : GSSException: {1}
  </MsgText>
  <Explanation>
    Excepción inesperada. La causa no se puede determinar inmediatamente.
  </Explanation>
  <UserResponse>
    Examine la excepción asociada para determinar la causa
  </UserResponse>
</Message>
<Message ID="SECJ9201W" severity="W" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.MultipleCredsFound" varFormat="Java">
    security.auth.kerberos.MultipleCredsFound=SECJ9201W: Se han encontrado varias credenciales Kerberos en el conjunto de credenciales privadas del sujeto; se utiliza la primera credencial del conjunto.
  </MsgText>
  <Explanation>
    Hay varias credenciales Kerberos en el conjunto de credenciales del sujeto. La autenticación continuará con la primera credencial del conjunto.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SECJ9320E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.ValidateException" varFormat="Java">
    security.auth.kerberos.ValidateException=SECJ9320E: La validación de la señal Kerberos ha lanzado la excepción siguiente: {0}
  </MsgText>
  <Explanation>
    La validación de la señal Kerberos ha lanzado una excepción no esperada
  </Explanation>
  <UserResponse>
    Examine la excepción asociada para determinar la causa.
  </UserResponse>
</Message>
<Message ID="SECJ9200E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.NoCredFound" varFormat="Java">
    security.auth.kerberos.NoCredFound=SECJ9200E: No se han encontrado credenciales Kerberos en el conjunto de credenciales de sujeto.
  </MsgText>
  <Explanation>
    No se ha encontrado ninguna credencial kerberos en el conjunto de credenciales privadas de sujeto JAAS.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SECJ9304E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.PrincipalMapNoDefaultRule" varFormat="Java">
    security.auth.kerberos.PrincipalMapNoDefaultRule=SECJ9304E: No se ha encontrado ninguna regla por omisión en el archivo de correlaciones &apos;&apos;{0}&apos;&apos;.
  </MsgText>
  <Explanation>
    No se ha encontrado una regla catch-all por omisión necesaria en el archivo de correlación de principales.
  </Explanation>
  <UserResponse>
    Asegúrese de que hay una regla catch-all por omisión en el archivo de correlaciones.
  </UserResponse>
</Message>
<Message ID="SECJ9322E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.RemPrincException" varFormat="Java">
    security.auth.kerberos.RemPrincException=SECJ9322E: La eliminación del principal del sujeto ha lanzado la excepción siguiente: {0}
  </MsgText>
  <Explanation>
    Elimina el principal del sujeto que ha lanzado una excepción no esperada.
  </Explanation>
  <UserResponse>
    Examine la excepción asociada para determinar la causa.
  </UserResponse>
</Message>
<Message ID="SECJ9206W" severity="W" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.noDelegatedCredentialsFound" varFormat="Java">
    security.auth.kerberos.noDelegatedCredentialsFound=SECJ9206W: No se han encontrado credenciales delegadas GSS para el usuario: {0}
  </MsgText>
  <Explanation>
    No se han encontrado credenciales delegadas GSS tras la validación de la petición de Kerberos.
  </Explanation>
  <UserResponse>
    Asegúrese de que el cliente tiene un ticket de Kerberos reenviable (TGT) y el servidor es de confianza para delegación.
  </UserResponse>
</Message>
<Message ID="SECJ9324E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.DestroyCredException" varFormat="Java">
    security.auth.kerberos.DestroyCredException=SECJ9324E: La destrucción de la credencial del sujeto ha lanzado la excepción siguiente: {0}
  </MsgText>
  <Explanation>
    Destruye la credencial pública del sujeto que ha lanzado una excepción no esperada.
  </Explanation>
  <UserResponse>
    Examine la excepción asociada para determinar la causa.
  </UserResponse>
</Message>
<Message ID="SECJ9332E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.complete.initnotcalled" varFormat="Java">
    security.auth.kerberos.complete.initnotcalled=SECJ9332E: No se ha llamado al método initSecContext() completo {0}
  </MsgText>
  <Explanation>
    El método initSecContext completo no está invocado.
  </Explanation>
  <UserResponse>
    Examine la excepción asociada para determinar la causa.
  </UserResponse>
</Message>
<Message ID="SECJ9203E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.CredDestroyed" varFormat="Java">
    security.auth.kerberos.CredDestroyed=SECJ9203E: El estado de la credencial no es válido.
  </MsgText>
  <Explanation>
    Se ha intentado acceder a una credencial destruida.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SECJ9314E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.unexpectedexception" varFormat="Java">
    security.auth.kerberos.unexpectedexception=SECJ9314E: Se ha producido una excepción inesperada al intentar ejecutar el método {0} : GSSException: {1}
  </MsgText>
  <Explanation>
    Excepción inesperada. La causa no se puede determinar inmediatamente.
  </Explanation>
  <UserResponse>
    Examine la excepción asociada para determinar la causa.
  </UserResponse>
</Message>
<Message ID="SECJ9302E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.PrincipalMapDuplicateDefaultRule" varFormat="Java">
    security.auth.kerberos.PrincipalMapDuplicateDefaultRule=SECJ9302E: Se ha encontrado una regla catch-all por omisión duplicada en el archivo de correlación &apos;&apos;{0}&apos;&apos; en la línea {1}.
  </MsgText>
  <Explanation>
    Se ha encontrado una regla catch-all por omisión en el archivo de correlación de principales.
  </Explanation>
  <UserResponse>
    Corrija el error en el archivo de correlación de principales.
  </UserResponse>
</Message>
<Message ID="SECJ9331E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.nullconstructor" varFormat="Java">
    security.auth.kerberos.nullconstructor=SECJ9331E: El constructor de la clase {0} no permite el valor nulo {1}
  </MsgText>
  <Explanation>
    La señal es nula.
  </Explanation>
  <UserResponse>
    Examine la excepción asociada para determinar la causa.
  </UserResponse>
</Message>
<Message ID="SECJ9202E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.GSSCredCopyFailed" varFormat="Java">
    security.auth.kerberos.GSSCredCopyFailed=SECJ9202E: Se ha producido un error en la operación de copia de una credencial GSS. Excepción GSS: {0}
  </MsgText>
  <Explanation>
    Se ha producido una excepción GSS al realizar una copia de una credencial GSS.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SECJ9330E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.init.nullcred" varFormat="Java">
    security.auth.kerberos.init.nullcred=SECJ9330E: La credencial es nula para el nombre de servicio GSS del destino {0} en el reino {1}
  </MsgText>
  <Explanation>
    La credencial es nual
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SECJ9312E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.PrincipalMapServerRootNotSet" varFormat="Java">
    security.auth.kerberos.PrincipalMapServerRootNotSet=SECJ9312E: No se ha establecido la propiedad del sistema &quot;server.root&quot;.
  </MsgText>
  <Explanation>
    No se ha establecido la propiedad del sistema &quot;server.root&quot;.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SECJ9308E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.PrincipalMapRuleMissingRHS" varFormat="Java">
    security.auth.kerberos.PrincipalMapRuleMissingRHS=SECJ9308E: A la regla de correlación de principales le falta el principal del lado derecho.
  </MsgText>
  <Explanation>
    La regla de correlación deben especificar un principal en el lado derecho del carácter de dos puntos.
  </Explanation>
  <UserResponse>
    Corrija la regla en el archivo de correlaciones.
  </UserResponse>
</Message>
<Message ID="SECJ9205E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.UserRegError" varFormat="Java">
    security.auth.kerberos.UserRegError=SECJ9205E: Error de registro de usuarios: {0}
  </MsgText>
  <Explanation>
    Se ha encontrado un error al acceder al registro de usuario.
  </Explanation>
  <UserResponse>
    Compruebe la configuración de registro de usuario.
  </UserResponse>
</Message>
<Message ID="SECJ9326E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.init.ctxnotestablished" varFormat="Java">
    security.auth.kerberos.init.ctxnotestablished=SECJ9326E: El contexto de seguridad no está establecido para GSSContext {0}
  </MsgText>
  <Explanation>
    El contexto de seguridad no está establecido.
  </Explanation>
  <UserResponse>
    Examine SystemErr.log para determinar el motivo.
  </UserResponse>
</Message>
<Message ID="SECJ9301E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.PrincipalMapError" varFormat="Java">
    security.auth.kerberos.PrincipalMapError=SECJ9301E: Error en el archivo de correlación de principales &apos;&apos;{0}&apos;&apos; en la línea {1}: {2}
  </MsgText>
  <Explanation>
    Se ha encontrado un error al leer el archivo de correlación de principales.
  </Explanation>
  <UserResponse>
    Corrija el error en el archivo de correlación de principales.
  </UserResponse>
</Message>
<Message ID="SECJ9207E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.gssUserNameIsNull" varFormat="Java">
    security.auth.kerberos.gssUserNameIsNull=SECJ9207E: El nombre de usuario GSS es nulo, {0}
  </MsgText>
  <Explanation>
    El nombre de usuario GSS es nulo.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SECJ9300E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.PrincipalMapNotFound" varFormat="Java">
    security.auth.kerberos.PrincipalMapNotFound=SECJ9300E: No se ha encontrado el archivo de correlación de principales &apos;&apos;{0}&apos;&apos; o no se puede acceder al mismo.
  </MsgText>
  <Explanation>
    El archivo de correlación de principales no se ha encontrado o no se puede acceder al mismo.
  </Explanation>
  <UserResponse>
    Asegúrese de que el archivo existe y que se puede acceder al mismo.
  </UserResponse>
</Message>
<Message ID="SECJ9309E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.PrincipalMapRuleMalformedLHS" varFormat="Java">
    security.auth.kerberos.PrincipalMapRuleMalformedLHS=SECJ9309E: Error en el lado izquierdo de la regla de correlación de principales.
  </MsgText>
  <Explanation>
    El lado izquierdo de la regla de correlación de principales debe ser uno de los siguientes: &apos;principal@reino&apos;, &apos;*@reino&apos; o &apos;*&apos;.
  </Explanation>
  <UserResponse>
    Corrija la regla en el archivo de correlaciones.
  </UserResponse>
</Message>
<Message ID="SECJ9310E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.PrincipalMapRuleMissingLHSPrincipal" varFormat="Java">
    security.auth.kerberos.PrincipalMapRuleMissingLHSPrincipal=SECJ9310E: A la regla de correlación de principales le falta el principal del lado izquierdo.
  </MsgText>
  <Explanation>
    La regla de correlación debe especificar un principal en el lado izquierdo del carácter de dos puntos.
  </Explanation>
  <UserResponse>
    Corrija la regla en el archivo de correlaciones.
  </UserResponse>
</Message>
<Message ID="SECJ9311E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.PrincipalMapRuleMissingLHSRealm" varFormat="Java">
    security.auth.kerberos.PrincipalMapRuleMissingLHSRealm=SECJ9311E: A la regla de correlación de principales le falta el reino del lado izquierdo.
  </MsgText>
  <Explanation>
    Una regla de correlación debe especificar un reino en el lado izquierdo del carácter de dos puntos.
  </Explanation>
  <UserResponse>
    Corrija la regla en el archivo de correlaciones.
  </UserResponse>
</Message>
<Message ID="SECJ9307E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.PrincipalMapRuleMissingLHS" varFormat="Java">
    security.auth.kerberos.PrincipalMapRuleMissingLHS=SECJ9307E: A la regla de correlación de principales le falta el reino y el principal del lado izquierdo.
  </MsgText>
  <Explanation>
    Una regla de correlación debe especificar un principal y un reino en el lado izquierdo del carácter de dos puntos.
  </Explanation>
  <UserResponse>
    Corrija la regla en el archivo de correlaciones.
  </UserResponse>
</Message>
<Message ID="SECJ9400W" severity="W" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.cannot.getSPNonClient" varFormat="Java">
    security.auth.kerberos.cannot.getSPNonClient=SECJ9400W: No se puede determinar el nombre principal de servicio Kerberos cuando se ejecuta en el cliente. Se devolverá un valor nulo.
  </MsgText>
  <Explanation>
    Cuando se invoca este método en el lado del cliente, no es posible determinar el nombre de principal del servicio de Kerberos y por lo tanto se devuelve un valor nulo.
  </Explanation>
  <UserResponse>
    No invoque este método en el lado del servidor.
  </UserResponse>
</Message>
<Message ID="SECJ9325E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.CredUtilsException" varFormat="Java">
    security.auth.kerberos.CredUtilsException=SECJ9325E: La creación de credencial ha lanzado la excepción siguiente: {0}
  </MsgText>
  <Explanation>
    Crear credencial ha lanzado una excepción no esperada.
  </Explanation>
  <UserResponse>
    Examine la excepción asociada para determinar la causa.
  </UserResponse>
</Message>
<Message ID="SECJ9313W" severity="W" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.RealmMismatch" varFormat="Java">
    security.auth.kerberos.RealmMismatch=SECJ9313W: El nombre de reino Kerberos especificado en el manejador de retorno de llamadas, {0}, no coincide con el nombre de reino Kerberos especificado en la configuración de Kerberos: {1} o el reino por omisión: {2}. El inicio de sesión continuará, ya que los módulos de inicio de sesión de WebSphere y Tivoli no comprueban los nombres de reinos.
  </MsgText>
  <Explanation>
    El nombre de reino de Kerberos especificado en el manejador de retorno de llamadas no coincide con el nombre de reino Kerberos o el nombre de reino por omisión, pero el inicio de sesión continuará de todas formas.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SECJ9323E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.RemCredException" varFormat="Java">
    security.auth.kerberos.RemCredException=SECJ9323E: La eliminación de la credencial pública del sujeto ha lanzado la excepción siguiente: {0}
  </MsgText>
  <Explanation>
    Elimina la credencial pública del sujeto que ha lanzado una excepción no esperada.
  </Explanation>
  <UserResponse>
    Examine la excepción asociada para determinar la causa.
  </UserResponse>
</Message>
<Message ID="SECJ9321E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.DoPrivException" varFormat="Java">
    security.auth.kerberos.DoPrivException=SECJ9321E: El método doPrivileged ha lanzado la excepción siguiente: {0}
  </MsgText>
  <Explanation>
    El método doPrivileged ha lanzado una excepción no esperada.
  </Explanation>
  <UserResponse>
    Examine la excepción asociada para determinar la causa.
  </UserResponse>
</Message>
<Message ID="SECJ9319E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.LoginException" varFormat="Java">
    security.auth.kerberos.LoginException=SECJ9319E: Se ha producido un error de inicio de sesión para el usuario {0}; la excepción es {1}
  </MsgText>
  <Explanation>
    Un inicio de sesión ha sido erróneo para el usuario.
  </Explanation>
  <UserResponse>
    Ninguna.
  </UserResponse>
</Message>
<Message ID="SECJ9315E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.exception" varFormat="Java">
    security.auth.kerberos.exception=SECJ9315E: Se ha producido una excepción inesperada al intentar ejecutar el método {0} : Excepción: {1}
  </MsgText>
  <Explanation>
    Excepción inesperada. La causa no se puede determinar inmediatamente.
  </Explanation>
  <UserResponse>
    Examine la excepción asociada para determinar la causa.
  </UserResponse>
</Message>
<Message ID="SECJ9306E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.PrincipalMapRuleMissingColon" varFormat="Java">
    security.auth.kerberos.PrincipalMapRuleMissingColon=SECJ9306E: A la regla de correlación de principales le falta el carácter de dos puntos necesario (&apos;:&apos;).
  </MsgText>
  <Explanation>
    Cada regla de correlación debe contener un carácter de dos puntos.
  </Explanation>
  <UserResponse>
    Corrija la regla en el archivo de correlaciones.
  </UserResponse>
</Message>
<Message ID="SECJ9305E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.PrincipalMapErrorsFound" varFormat="Java">
    security.auth.kerberos.PrincipalMapErrorsFound=SECJ9305E: Se han encontrado errores al procesar el archivo de correlaciones &apos;&apos;{0}&apos;&apos;.
  </MsgText>
  <Explanation>
    Se han encontrado errores al procesar el archivo de correlaciones.
  </Explanation>
  <UserResponse>
    Corrija el error en el archivo de correlación de principales.
  </UserResponse>
</Message>
<Message ID="SECJ9303E" severity="E" prefix="yes">
  <MsgText pgmKey="security.auth.kerberos.PrincipalMapIOException" varFormat="Java">
    security.auth.kerberos.PrincipalMapIOException=SECJ9303E: Se ha capturado una excepción IOException al leer el archivo de correlación de principales &apos;&apos;{0}&apos;&apos;.
  </MsgText>
  <Explanation>
    Se ha encontrado un excepción de entrada/salida al leer el archivo de correlación de principales.
  </Explanation>
  <UserResponse>
    Asegúrese de que el archivo existe y que se puede leer.
  </UserResponse>
</Message>
<!-- END MESSAGES -->
</TMSSource>
